docs/server/commands.rst

```diff
@@ -413,6 +416,10 @@ server. Use :option:`--refresh` to also update existing translations that have
 been changed, besides adding any new translation. To completely remove the TM
 and rebuild it adding all existing translations use :option:`--rebuild`.
 
+If no specific TM server is specified using :option:`--tm`, then the default
+``default`` TM will be used. If the specified TM server doesn't exist it will
```

Comment:
Doesn't TM default to local TM, hence `local`? If so, you might wanna check other occurrences of `default` in the docs.